data_steward/bq_utils.py

```diff
@@ -739,6 +739,31 @@ def create_dataset(
     return insert_result
 
 
+def get_sql_row_expr(row: dict, fields: list):
+    """
+    Get appropriate SQL expression for a row based on fields spec
+
+    :param row: csv row with untyped values
```

Comment:
What does untyped mean here - are they arbitrary types?

Reply:
Thanks, untyped was the wrong word- values are represented as strings. I made the docstring more explicit.